common.mk

```diff
@@ -731,6 +731,15 @@ ifeq ($(CC_VENDOR),clang)
 CWARNFLAGS += -Wno-tautological-compare -Wno-pass-failed
 endif
 
+# Disable other annoying warnings.
+ifeq ($(CC_VENDOR),clang)
+CWARNFLAGS +=
+else
+ifeq ($(CC_VENDOR),gcc)
+CWARNFLAGS += -Wno-maybe-uninitialized -Wno-comment
+endif
+endif
+
```

Comment:
@devinamatthews Not sure I've ever used these options. Is there any situation where either of these could come back to bite us?

Reply:
`-Wno-maybe-unitialized` could in principle mask an actual initialization issue, but in my experience these are mostly false positives in BLIS.

Reply:
Okay, thanks.